Source/Core/Core/IOS/Network/KD/NetKDRequest.cpp

```diff
@@ -21,6 +22,126 @@
 
 namespace IOS::HLE
 {
+namespace
+{
+enum class HardwareModel : u8
+{
+  RVT = 0,
+  RVV = 0,
+  RVL = 1,
+  RVD = 2,
+  Unknown = 7
+};
+
+u8 GetAreaCode(std::string_view area)
+{
+  static constexpr std::array<std::pair<std::string_view, u8>, 13> regions{{
```

Comment:
Doesn't this need `utility` (for `std::pair`) and `string_view` (for, well...`std::string_view`) included?